package test

fun nothing(): Array<out Number> = throw Exception()
